src/test/java/com/zendesk/maxwell/row/RowMapTest.java

```diff
@@ -55,4 +56,198 @@ public void testTimestampConversion() throws Exception {
+		long timestampMillis = 1496712943487L;
+
+		Position position = new Position(new BinlogPosition(1L, "binlog-0001"), 0L);
+		RowMap rowMap = new RowMap("insert", "MyDatabase", "MyTable", timestampMillis,
+				null, position);
+
+		rowMap.putData("id", "9001");
+		rowMap.putData("first_name", "foo");
+		rowMap.putData("middle_name", "buzz");
+		rowMap.putData("last_name", "bar");
+		rowMap.putData("salary", "4000");
+		rowMap.putData("department", "science");
+        rowMap.putData("rawJSON", new RawJSONString("{\"database\":\"MyDatabase\",\"table\":\"User\",\"type\":\"insert\",\"ts\":1486439516,\"xid\":5694,\"commit\":true,\"data\":{\"UserID\":20,\"FirstName\":\"Fiz\",\"MiddleName\":null,\"LastName\":\"Buz\",\"Version\":1486703131000}}\n"));
+		MaxwellOutputConfig outputConfig = getMaxwellOutputConfig();
+
+		Assert.assertEquals("{\"database\":\"MyDatabase\",\"table\":\"MyTable\",\"type\":\"insert\",\"ts\":1496712943,\"position\":\"binlog-0001:1\",\"gtid\":null,\"data\":{\"id\":\"9001\",\"salary\":\"4000\",\"department\":\"science\",\"rawJSON\":{\"database\":\"MyDatabase\",\"table\":\"User\",\"type\":\"insert\",\"ts\":1486439516,\"xid\":5694,\"commit\":true,\"data\":{\"UserID\":20,\"FirstName\":\"Fiz\",\"MiddleName\":null,\"LastName\":\"Buz\",\"Version\":1486703131000}}\n" +
+				"}}", rowMap.toJSON(outputConfig));
+	}
+
+	@Test
+	public void testToJSONWithListData() throws IOException {
+		long timestampMillis = 1496712943487L;
+
+		Position position = new Position(new BinlogPosition(1L, "binlog-0001"), 0L);
+		RowMap rowMap = new RowMap("insert", "MyDatabase", "MyTable", timestampMillis,
+				null, position);
+
+		rowMap.putData("id", "9001");
+		rowMap.putData("first_name", "foo");
+		rowMap.putData("middle_name", "buzz");
+		rowMap.putData("last_name", "bar");
+		rowMap.putData("salary", "4000");
+		rowMap.putData("department", "science");
+		rowMap.putData("interests", Arrays.asList("classical music", "hiking", "biking", "programming"));
+		MaxwellOutputConfig outputConfig = getMaxwellOutputConfig();
+
+		Assert.assertEquals("{\"database\":\"MyDatabase\",\"table\":\"MyTable\",\"type\":\"insert\",\"ts\":1496712943,\"position\":\"binlog-0001:1\",\"gtid\":null,\"data\":{\"id\":\"9001\",\"salary\":\"4000\",\"department\":\"science\",\"interests\":[\"classical music\",\"hiking\",\"biking\",\"programming\"]}}", rowMap.toJSON(outputConfig));
+	}
+
+	private MaxwellOutputConfig getMaxwellOutputConfig() {
+		MaxwellOutputConfig outputConfig = new MaxwellOutputConfig();
+
+
+
+		List<Pattern> patterns = new ArrayList<>();
+		Pattern pattern = Pattern.compile("^.*name.*$");
+		patterns.add(pattern);
+
+		outputConfig.includesBinlogPosition = true;
+		outputConfig.includesCommitInfo = true;
+		outputConfig.includesGtidPosition = true;
+		outputConfig.includesServerId = true;
+		outputConfig.includesThreadId = true;
+		outputConfig.includesNulls = true;
+		outputConfig.excludeColumns = patterns;
```

Comment:
Do these tests actually care about all of these settings? It's fine to use the defaults unless you actually need different behaviour.